crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -147,3 +172,18 @@ where
             .collect()
     }
 }
+
+impl<TYPES, PUBKEY: SignatureKey + 'static> GeneralStaticCommittee<TYPES, PUBKEY>
+where
+    TYPES: NodeType<SignatureKey = PUBKEY, ElectionConfigType = StaticElectionConfig>,
+{
+    /// get the non-staked builder nodes
+    pub fn non_staked_nodes_count(&self) -> usize {
+        self.committee_nodes_without_stake.as_ref().unwrap().len()
+    }
+
+    /// get all the non-staked nodes
+    pub fn get_non_staked_nodes(&self) -> Vec<PUBKEY> {
```

Comment:
Is this function being used?

Reply:
Not currently, but would be used later.

Reply:
Forgot to mention, we'll have to use this function in hotshot shot here: https://github.com/EspressoSystems/HotShot/blob/main/crates/task-impls/src/network.rs#L375